pkg/oauth/client.go

```diff
@@ -0,0 +1,98 @@
+package oauth
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/baseclient"
+)
+
+const (
+	baseUrl = "https://api.mercadopago.com/oauth/token"
+	urlAuth = "https://auth.mercadopago.com/authorization"
```

Comment:
Para manter o padrão, o que acha?

```suggestion
	authUrl = "https://auth.mercadopago.com/authorization"
```

Reply:
O padrão dos outros clients é sempre iniciar com url... Exemplo de payment: **urlSearch**

Reply:
o que de fato deveria mudar seria o baseUrl para **urlBase**